js/src/framework.ts

```diff
@@ -108,14 +144,23 @@ globalThis._evals = {};
 export async function Eval<Input, Output, Expected>(
   name: string,
   evaluator: Evaluator<Input, Output, Expected>
-): Promise<void | ExperimentSummary> {
+): Promise<ExperimentSummary> {
   const evalName = makeEvalName(name, evaluator.experimentName);
   if (_evals[evalName]) {
     throw new Error(`Evaluator ${evalName} already exists`);
   }
   if (globalThis._lazy_load) {
     _evals[evalName] = { evalName, projectName: name, ...evaluator };
-    return;
+    // Better to return this empty object than have an annoying-to-use signature
```

Comment:
Isn't the standard solution here to return `undefined` or `null`?

Reply:
Generally, yes, but our users _never_ expect this to be the case (`_lazy_load` is not something they can ever trigger). So I don't think it makes sense to pollute the function's type.